package.json

```
@@ -469,6 +474,11 @@
           "when": "editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive",
           "group": "objectscript@3"
         },
+        {
+          "command": "vscode-objectscript.refreshLocalFile",
+          "when": "editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive",
```

```suggestion
          "when": "resourceScheme == file && editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive",
```
Otherwise I think it would appear for isfs documents.